src/openrct2/Context.cpp

```diff
@@ -395,7 +395,7 @@ namespace OpenRCT2
 
         bool LoadParkFromStream(IStream * stream, const std::string &path, bool loadTitleScreenFirstOnFail) final override
         {
-            ClassifiedFileInfo info;
+            ClassifiedFileInfo info{};
```

Comment:
It would be better to give it default values in the struct definition.

Reply:
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!
# LOL PINEAPPLE SEX!